### ISSUE TITLE
How to determine which conferences should be abbreviated with -HLT?

### ISSUE THREAD

**Issue by nschneid** [acl-org/acl-anthology#453]
#413 removed HLT as a co-venue for 2007+. The Zotero translator rewrite (https://github.com/zotero/translators/pull/1911) has been assuming that the venues for a publication can be used to determine the abbreviated conference name, e.g. "NAACL-HLT", but now this no longer holds. Is there another way to find out from the paper metadata which conferences are HLT-affiliated?

**Comment by mjpost** [comment]
My understanding is that HLT was a government-run language conference that was jointly held with NAACL for a number of years. In 2007, HLT ceased to exist, but its name was incorporated into the conference, likely to reflect the work that was done by the HLT folks. So NAACL conferences continue to include this name, as did ACL [2008](https://aclweb.org/anthology/events/acl-2008/) and [2011](https://aclweb.org/anthology/events/acl-2011/) (but not [2005](https://aclweb.org/anthology/events/acl-2005/)). 

So unfortunately, I don't think this is easy from the metadata. You could pull information from [the joint venues map](https://github.com/acl-org/acl-anthology/blob/master/data/yaml/venues_joint_map.yaml), but that might get changed at some point.

@orambow may be able to say more.

**Comment by nschneid** [comment]
I guess a heuristic that would work for now is to consult the full title: all the *-HLT conferences (I think) contain `Human Language Technolog(y|ies)` or `HLT` in the title.

@guyaglionby, is it worth implementing this in the Zotero importer?

**Comment by davidweichiang** [comment]
I think it would be useful for the yaml files somewhere to list the short name of every conference. Some of them are very idiosyncratic, like "ACL-08: HLT," and probably can't be determined algorithmically from the long names. In some cases, the short name was very intentionally chosen, with long discussions behind these short names about whether there should be dashes or spaces or colons, and we should try to respect that. In some cases, however, the short names are not consistent, so I'm not sure which to choose.

**Comment by mjpost** [comment]
Yes, we will need this information to generate the new IDs for next year. We should have a place for it. Current W* workshops/conferences will have short codes added as soon as they have a 2020+ ingestion.
